include/GPM/Shape3D/Quad.hpp

```diff
@@ -0,0 +1,101 @@
+﻿/*
+ * Copyright (C) 2021 Amara Sami, Dallard Thomas, Nardone William, Six Jonathan
+ * This file is subject to the LGNU license terms in the LICENSE file
+ * found in the top-level directory of this distribution.
+ */
+
+#pragma once
+
+#include "../Vector3.hpp"
+#include "Plane.hpp"
+#include "../Referential.hpp"
+#include "../Tools.hpp"
+
+#include <limits>
+
+namespace GPM
+{
+    #define QUAD_OUTCODE_INSIDE   0  // 0000
```

Comment:
I don't see the use of these macros.
Also, macros are available outside of namespaces.
We could surely put it in an inlined function.